Cargo.toml

```diff
@@ -79,7 +79,7 @@ assert_cmd = "1.0.1"
 [build-dependencies]
 clap = { version = "2.33", optional = true }
 liquid = { version = "0.20", optional = true }
-lazy_static = { version = "1.4", optional = true }
 
 [profile.release]
 lto = true
+codegen-units = 1
```

Comment:
Could you please explain what this does?

Reply:
https://doc.rust-lang.org/nightly/rustc/codegen-options/index.html#codegen-units
explain it better than me.